helm/track.yml

```diff
@@ -0,0 +1,14 @@
+slug: helm
+id: jzui2fhkbxbq
```

Comment:
I think you will need to delete the ID here and push to `templates` in production to create the resource first. I can show you how, when we're done with the copy changes.